code/game/machinery/computer/abnormality_work.dm

```diff
@@ -13,6 +13,8 @@
 	var/meltdown_time = 0
 	/// Can the abnormality even meltdown?
 	var/can_meltdown = TRUE
+	/// Has the Abnormality reached max Understanding?
+	var/fully_understood = FALSE
```

Comment:
Why is it here?

Reply:
The purpose of this was to make sure I don't hit the "gift chance multiplier" multiple times

Reply:
This one is in the console variables and is unused.

Reply:
I think I moved around it and have removed it